requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Base requirements for apel
 
-MySQL-python
-python-ldap<3.4.0  # python-ldap-3.4.0 dropped support for Python 2
-iso8601
+mysqlclient==2.1.1
+python-ldap
+iso8601==1.1.0
 dirq
+future
```

Comment:
Can you add a comment that this provides `future`. Is there an rpm package easily available that can be added to the SPEC file?

Reply:
Sure. Added comments for future explaining why we are using this in short now. And also added the relevant rpm in SPEC file to make it work with RPM commands itself.